x-pack/plugins/enterprise_search/public/applications/__mocks__/index.ts

```diff
@@ -23,3 +23,5 @@ export { mountWithIntl } from './mount_with_i18n.mock';
 export { shallowWithIntl } from './shallow_with_i18n.mock';
 export { rerender } from './enzyme_rerender.mock';
 // Note: shallow_useeffect must be imported directly as a file
+
+export { expectedAsyncError } from './expected_async_error';
```

Comment:
Just btw, I know I'm getting a little loosey-goosey here with what should go into the `__mocks__` folder, especially since this is definitely strictly a test helper and not a mock 😅 I'm definitely happy to revisit at some point if we just want a generic test helper folder, or if we want to split this up later at some point

Reply:
I'm not concerned about it.